### ISSUE TITLE
PDFs from file links are being automatically downloaded instead of opened in Brave [Desktop]

### ISSUE THREAD

**Issue by GitHunter0** [brave/brave-browser#11384]
Regardless of any setting, PDFs from file links are being automatically downloaded instead of opened in Brave. In Firefox it works as desired.
For example, in the file attached, that happens to any URL pointing to PDF.
[+ R Workflow.pdf](https://github.com/brave/brave-browser/files/5115293/%2B.R.Workflow.pdf)

It would be very nice to have that issue fixed. Brave is an awesome browser. Thank you.

**Comment by sameyepatch** [comment]
I'm using Brave in Fedora 32, getting the same behavior. It gets downloaded instead of opening it in the browser. So it's happening in Linux as well.

Brave V1.13.82

**Comment by GitHunter0** [comment]
@rebron , Google Chrome is presenting the same behavior, so could it be Chromium the root of this issue?

**Comment by bsclifton** [comment]
@GitHunter0 yes - that's definitely possible. We're using "PDFium" from Chromium so I think the behavior in Brave isn't diverging

**Comment by ddoherty03** [comment]
Same here.  Very annoying. Ubuntu 20.04.

**Comment by AlterMind64** [comment]
Hi there, was any correction of any kind implemented since the issue was raised ?

I have a very similar issue, extensively described in Brave's Cummunity Support forum as [https://community.brave.com/t/clicking-on-pdf-links-will-prompt-to-save-file-instead-of-allowing-to-open-it/268853](url)

In a nutshell : clicking on some links to PDF with name contains 'spaces' as in in my case (while '+' and '.' in this thread) will not open in Brave nor launch the default system PDF viewer in Windows 10, but instead download it immediately (prompting or not for the location depending on the 'PDF documents' handling setting). This odd behaviour is **not** observed with Chrome, Firefox or Edge which all work fine and as expected -- only tested on Windows 10, not on Linux. 

I have the issue with Brave both [my link to PDF here](http://www.pierpont.be/download/i/mark_dl/u/4010359942/4637741192/GdP%20-%20Greenfee%202020.pdf) (found on [this web page](http://www.pierpont.be/greenfee/4563568292) as with [OP's link to PDF here](https://github.com/brave/brave-browser/files/5115293/%2B.R.Workflow.pdf).

But as I do **not** have the issue with Chrome (with none of the two PDFs linked), I was wondering if we're facing here the same problem (issue/bug) or if a separate bug report case ought to be created ?  Thanks for your feedback.

**Comment by AlterMind64** [comment]
> It's not a fix but you can try this [extension ](https://chrome.google.com/webstore/detail/pdf-viewer/oemmndcbldboiebfnladdacbdfmadadm)it solved it for me

In your implementation, does it offer you the choice to view/download or is it necessarily opening it ?

**Comment by dabupr** [comment]
> In your implementation, does it offer you the choice to view/download or is it necessarily opening it ?

It always opens the pdf, but once it's open you can download it by a button in the top right
